mmv1/products/apigee/api.yaml

```
@@ -231,6 +231,13 @@ objects:
         message: 'message'
     description: |
       An `Environment` in Apigee.
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
+      exclude: false
+      method_name_separator: ':'
+      parent_resource_attribute: 'env_id'
+      import_format: ["{{org_id}}/environments/{{name}}", "{{name}}"]
```

This `%` will indicate that the org_id can contain slashes, which should resolve the problem. (Currently the import_format regex excludes the "organization/" at the start of the org id.)

```suggestion
      import_format: ["{{%org_id}}/environments/{{name}}", "{{name}}"]
```